lib/bolt/project.rb

```diff
@@ -209,12 +209,5 @@ def validate
         Bolt::Logger.warn("missing_project_name", message)
       end
     end
-
-    def check_deprecated_file
```

Comment:
Can this get moved to `Bolt::Project#validate`?

Reply:
I think this could probably be removed altogether since `project.yaml` was only present in 2.8.0 and 2.9.0 and was listed as experimental at the time.